app/Theme/Plugins.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Dxw\MyTheme\Theme;
+
+use Dxw\Iguana\Registerable;
+
+class Plugins implements Registerable
+{
+    protected $required;
+    protected $abspath;
+
+    public function __construct(array $required = [], $abspath = null)
+    {
+        $this->required = $required;
+        $this->abspath = $abspath ?? ABSPATH;
+    }
+
+    public function register()
+    {
+        add_action('after_switch_theme', [$this, 'checkDependencies']);
+    }
+
+    public function checkDependencies()
+    {
+        $pluginsToActivate = $this->findPluginsToActivate();
+        if (!empty($pluginsToActivate) && !function_exists('get_plugins')) {
```

Comment:
Are these conditionals about the same thing?

Would it make sense to instead exit early if there are no plugins to activate, and then separately ensure that /includes/plugin is loaded. 

Would it also be better to check for `get_plugin_data` since that's the method we're using?